pyvista/plotting/itkplotter.py

```diff
@@ -13,32 +10,109 @@
 except ImportError:
     pass
 
+
+def version_tuple(v):
+    """Converts a version string to a tuple"""
+    return tuple(map(int, (v.split("."))))
+
+
+def meets_version(va, vb):
+    """Check if a version string meets a minimum version.
+
+    Parameters
+    ----------
+    va : str
+        Version string.  For example ``'0.25.1'``.
+
+    va : str
+        Version string.  For example ``'0.25.2'``.
+
+    Returns
+    -------
+    newer : bool
+        True if version ``va`` is greater or equal to version ``vb``.
+
+    Examples
+    --------
+    >>> meets_version('0.25.1', '0.25.2')
+    False
+
+    >>> meets_version('0.26.0', '0.25.2')
+    True
+    """
+    va = version_tuple(va)
+    vb = version_tuple(vb)
+
+    for i in range(3):
+        if va[i] > vb[i]:
+            print('ge')
+            return True
+        elif va[i] < vb[i]:
+            print('le')
+            return False
+    return True
```

Comment:
sorry to keep suggesting changes, but I think we should put this in `scooby`, make a release, and bump the dependency in this PR.

You up for this @akaszynski? Or should I jump on it?

Reply:
I've got no issue with the changes, and it sounds like a good idea since `scooby` is already a requirement and this doesn't really belong here.

Since you're more familiar with `scooby`, would you mind adding it?  Once it's added, I'll modify this PR.

Check the latest commit, I removed the print statement there.

Reply:
Awesome, I'll put this in `scooby` asap